components/CountyGuidelines.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="CountyGuidelines">
+    <div v-for="(item, i) in info[county].info" :key="i">
+      <v-card class="guidelines">
+        <p class="Header">
+          {{ item.header }}
+        </p>
+        <!-- <p class="time-updated">
```

Comment:
Maybe future proof this by adding it in and using a v-if to show or not to show. Possibly add a test date in the json to test this field.

Reply:
added this on latest commit